python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -719,7 +722,9 @@ def getRefFluxField(schema, filterName=None):
     if not isinstance(schema, afwTable.Schema):
         raise RuntimeError("schema=%s is not a schema" % (schema,))
     if filterName:
-        fluxFieldList = [filterName + "_camFlux", filterName + "_flux"]
+        # NOTE: include "camFlux" in the list for the case where
+        # `defaultFilter` was specified in the refcat loader config.
+        fluxFieldList = [filterName + "_camFlux", filterName + "_flux", "camFlux"]
```

Comment:
This code will silently use "camFlux" even if you have a filter map, but you misspell `filterName` or the associated entry in the filter map. That worries me.

If there is a reasonable way to test if you have a filter map, then I suggest the following, instead:
```
if filterName and a filter map exists:
    fluxFieldList = [filterName + "_camFlux", filterName + "_flux"]
else:
    fluxFieldList = ["camFlux"]
```